code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -273,7 +291,9 @@
 	name = "box of donk-pockets"
 	desc = "<B>Instructions:</B> <I>Heat in microwave. Product will cool if not eaten within seven minutes.</I>"
 	icon_state = "donk_kit"
-	startswith = list(/obj/item/chems/food/donkpocket = 6)
+	startswith = list((/obj/item/chems/food/donkpocket = 6)
```

Comment:
Extra `(`

Reply:
I love how I managed to break the code by fixing a merge conflict.. :D